src/components/WeeklyComponent.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCurrentCity } from "@/contexts/currentCity";
+import { WeatherDay } from "@/types";
+import styles from "./WeeklyComponent.module.scss";
```

Comment:
I think the path here would be like this?
```suggestion
import styles from "../style/components/WeeklyComponent.module.scss";
```